.github/workflows/testing.yml

```diff
@@ -40,7 +42,7 @@ jobs:
 
     - name: Generate Helm values file
       run: |
-        ./Toolsmith/helm-values/aws-cicd-pr-deployment.yaml.sh ${{ env.PR_NUMBER }} > pr-deployment.yaml
+        ./Toolsmith/helm-values/aws-cicd-testing-pr-deployment.yaml.sh ${{ env.PR_NUMBER }} > pr-deployment.yaml
```

Comment:
Why do you think we need a separate YAML file? I guess you didn't touch the original because you didn't want to break the build... we just need to remember to remove the original then once this PR is merged.

Reply:
yes, this would break the build for other branches, unfortunately. yes, we need to fix this post merge.